bestofjs/bestofjs
# WIP: Add specific OG image for project list pages

## Goal
Add a specific OG image for project list pages `/projects`.

More details about this PR can be found in the issue: https://github.com/bestofjs/bestofjs/issues/210
## How to test

- Run `pnpm install` at root of repository
- Pull this branch and checkout locally
- Go to [http://localhost:3000/api/og/projects](http://localhost:3000/api/og/projects) - there should be a new OG image that is similar to what is seen at [http://localhost:3000/projects](http://localhost:3000/projects)
- try adding `query` and `tags` to the end of `projects/`. For example: `?query=tool&tags=cli` - The OG image should display the first 3 projects based on the search params.
